internal/cmd/connect/install_utils.go

```diff
@@ -227,7 +229,7 @@ func standardWorkerConfigLocations(ins *installation) ([]WorkerConfig, error) {
 		if utils.DoesPathExist(confluentCurrentFile) {
 			confluentCurrentContent, err := os.ReadFile(confluentCurrentFile)
 			if err != nil {
-				return nil, errors.Errorf("failed to read possible $CONFLUENT_CURRENT file %s: %v", confluentCurrentFile, err)
+				return nil, errors.Wrapf(err, `failed to read possible $CONFLUENT_CURRENT file "%s"`, confluentCurrentFile)
```

Comment:
This seems like an artifact of the old `confluent local` commands. Not sure we need it anymore, but I'd double check.